package.json

```diff
@@ -46,7 +47,12 @@
   },
   "browserify": {
     "transform": [
-      ["babelify", {"stage": 0}]
+      [
```

Comment:
Can you remove this change & squish the 3 commits into one? Other than this, everything looks ready to be merged :+1: